docs/src/main/sphinx/release/release-411.md

```diff
@@ -0,0 +1,59 @@
+# Release 411 (15 Mar 2023)
+
+## General
+
+* Add spilled data size to query statistics. ({issue}`16442`)
+* Add {func}`sinh` function. ({issue}`16494`)
+* Add support for a `GRACE PERIOD` clause in the `CREATE MATERIALIZED VIEW`
+  task. This is a backwards incompatible change, and the previous behavior can
+  temporarily be restored with the `legacy.materialized-view-grace-period`
+  configuration property or the `legacy_materialized_view_grace_period` session
+  property. ({issue}`15842`)
+* Add `CANCELING`, `ABORTING`, and `FAILING` task statuses for tasks which are
+  waiting on running drivers to fully terminate. ({issue}`15478`)
+* Improve error messages for exceptions thrown by a connector while querying
+  metadata. ({issue}`16497`)
+* Fix generation of IDs for `keyvalue()` methods in the
+  [JSON path](json-path-language). Previously, separate IDs were generated for
+  each member. ({issue}`16482`)
+
+## Delta Lake connector
+
+* Add support for inserting into tables that have
+  [column invariants](https://github.com/delta-io/delta/blob/master/PROTOCOL.md#column-invariants). ({issue}`16136`)
```

Comment:
Is there no link to the docs instead of the source code?

Reply:
They have https://docs.delta.io/, but "column invariants" doesn't exist in the page.